tests/spec/fs.stat.spec.js

```diff
@@ -91,4 +91,32 @@ describe('fs.stat', function() {
       });
     });
   });
+
+  it('(promise) should be a function', function() {
+    var fs = util.fs();
+    expect(fs.promises.stat).to.be.a('function');
+  });
+  
+  it('should return a promise', function() {
+    var fs = util.fs();
+    expect(fs.promises.stat()).to.be.a('Promise');
+  });
+  
+  it('(promise) should return a stat object if file exists', function(){
+    var fs = util.fs();
+
+    return fs.promises.stat('/')
+      .then(result => { 
+        expect(result).to.exist;
+        expect(result['node']).to.be.a('string');
```

Comment:
Doing `result['node']` vs. `result.node` is fine, but the latter is more common.  If we're doing `[]` indexing into the object above, I can see why you copied the same style.  In general, I would prefer `result.node`, unless the key name is reserved, or being assembled as a variable.  Same below.